docs/directory/built-ins.mdx

```diff
@@ -40,10 +33,10 @@ For example: `ds.object({ "type": "identity", "key": "euang@acmecorp.com" })` wi
 
 ```js
 ds.relation({
-  "object_id": "<object-id>",
```

Comment:
I'm pretty sure the signature of `ds.relation` is out of date - I believe we have `"subject": { "type": "x", "key": "y" }`, etc, just like the signature for `check_relation`